### ISSUE TITLE
SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by janbaykara** [RasCarlito/axios-cache-adapter#4]
This could well be me noob'ing with babel but this shows up when I try to use your pkg (`import { setupCache } from 'axios-cache-adapter'`) and run with `nodemon server/www.js --exec babel-node`, I get this error:
```
/Users/Jan/Sites/rssmag/node_modules/lodash-es/omit.js:1
(function (exports, require, module, __filename, __dirname) { import arrayMap from './_arrayMap.js';
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:588:28)
    at Module._extensions..js (module.js:635:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Jan/Sites/rssmag/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
```

**Comment by RasCarlito** [comment]
Hi @janbaykara !

Thanks for reporting the bug. I'll switch to `lodash` instead of `lodash-es` for it to work in an environment without ESM support. I haven't tried running my script in nodejs very much. I definitly needs more testing.

**Comment by RasCarlito** [comment]
Released a patch hoping to fix this [v1.5.2](https://github.com/RasCarlito/axios-cache-adapter/releases/tag/1.5.2)
